src/Quasar/Advanced/Types.purs

```diff
@@ -464,6 +464,37 @@ instance encodeJsonProvider ∷ EncodeJson Provider where
     ~> "client_id" := Newtype.unwrap obj.clientId
     ~> "openid_configuration" := OpenIDConfiguration obj.openIDConfiguration
     ~> jsonEmptyObject
+    
+data LicenseStatus = LicenseValid
+
+decodeLicenseStatus ∷ Json → Either String LicenseStatus
+decodeLicenseStatus =
+  (if _ then Right LicenseValid else Left $ "status != " <> show licenseValid)
+    <<< eq licenseValid
+    <=< decodeJson
+  where
+  licenseValid = "LICENSE_VALID"
+
+type LicenseInfo'
+  = { expirationDate ∷ String
```

Comment:
@beckyconning `purescript-js-date` shouldn't really be used for parsing if it can be avoided, you should be able to `unformat` this format with https://github.com/slamdata/purescript-formatters though.